apps/judicial-system/api/src/app/modules/backend/backend.service.ts

```diff
@@ -24,6 +24,16 @@ import {
   SignatureConfirmationResponse,
 } from '../case'
 import { CaseListEntry, CaseStatistics } from '../case-list'
+import {
```

Comment:
Please consider using barrel imports in the api project to be consistent with other imports.

Reply:
Ahh sorry, I keep forgetting this pattern here 🙈 . I will fix this in the follow-up work [here](https://app.asana.com/1/203394141643832/project/1199153462262248/task/1210998427054268)